nam/models/base.py

```diff
@@ -259,3 +275,20 @@ def _mse_loss(self, preds, targets, pre_emph_coef: Optional[float] = None):
                 z[..., 1:] - pre_emph_coef * z[..., :-1] for z in (preds, targets)
             ]
         return nn.MSELoss()(preds, targets)
+
+    def _mrstft_loss(self, preds: torch.Tensor, targets: torch.Tensor) -> torch.Tensor:
```

Comment:
Is there a reason that this has to be a method of  `Model`? If not, i'd rather see it implemented in `losses.py` where the others are.

Reply:
Sure, but we need to instantiate the MultiResolutionSTFTLoss and store it somewhere - I think instantiating it in every loss function call is not a good idea. All the previous losses are stateless and therefore this needs different handling. https://github.com/csteinmetz1/auraloss#usage

Another alternative would be to have AuraLoss class defined in losses.py and its instance would be stored in Model.

Reply:
Hmm. Interesting.

On measure, I think I'd rather have this and I'm happy to explore it more deeply on my own time :)